### ISSUE TITLE
2018.08.1 One contact is not editable

### ISSUE THREAD

**Issue by ignoreigor** [tine20/tine20#6874]
In version 2018.08.1 I have a strange behaviour:  I found a contact, that is not editable. The dialog is opening, data is loaded and shown (complete at the first view), but it seems that loading is never finished, controls don't become active. But at this time, it's the only contact that shows this bug. There are no JS-errors and there are no server logs - neither in tine.log at loglevel 3 nor in Apache's error.log.
With that contact it's reproduceable, but I have no idea what difference is causing that issue. I thought about umlauts or links , but there are other contactitems with umlauts or links, that work. 

I will have a closer look on this...

**Comment by pschuele** [comment]
hi ignoreigor (nice nickname),
does the contact have relations to other records? this sounds like a bug in the js frontend to me.
maybe you'll see more (in the javascript console), when you set this in your config.inc.php:

```php
'buildtype' => 'DEBUG',
```

**Comment by ignoreigor** [comment]
Hi Philipp,
if you like my nickname, you would love my password ;-)
I changed the buildtype to debug and turned on tine20's debug-console. The debug-console is empty, pushing "run" says "(no return)". Of course there are objects in the inspector-tabs and data stores.
Awareness while writing: Aaah, I see "trap errors" pushes the messages to the browsers console. Well, there is logged a loading of the objects data, but as I can see, no errors. Meanwhile Tine20-server logs method  Tinebase_Server_Json::Addressbook.getContact.

You are right: the item is linked to another record. 
```
| id                                       | rel_id                                   | own_model                 | own_backend | own_id                                   | related_degree | related_model          | related_backend | related_id                               | type | remark | created_by                               | creation_time       | last_modified_by | last_modified_time | is_deleted | deleted_by | deleted_time | seq |

| 944c0325a83c34ae8518754a3ae2a616a9b2105a | 145190fa0fc7eb9b9c29a8a1ee6a9f04fe7d68a0 | Addressbook_Model_Contact | Sql         | 5ada8fe27da45339cde2d753032b0f91720fb101 | sibling        | Filemanager_Model_Node | Sql             | 74255bb7f04f618b39fb45c41029170fa511d5bb |      | NULL   | 5d6aa4c9bc8929e6410f1440ee2bfe36072f7c94 | 2013-04-30 16:07:43 | NULL             | NULL               |          0 | NULL       | NULL         |   0 |
```
I can't find any object "74255bb7f04f618b39fb45c41029170fa511d5bb" neither in tine20_tree_nodes, tine20_container nor in tine20_filemanager_downloadlink. Maybe an expired downloadlink?! Can't find this ID in my sql-dumps on any other place except the relation-set.

After deleting this relation via update tine20_relations set is_deleted = 1 where own_id = '5ada8fe27da45339cde2d753032b0f91720fb101' , the contact is opening again. Maybe calling the CLI-method cleanRelations would have done the same?!

In summary: the record was linked to another record that I could not find or doesn't exists. Trying to find this relation while opening the contact record could have aborted the loading.

Thanks for your help.

**Comment by pschuele** [comment]
ok, thanks for the extensive feedback. and good, that you could help yourself.

but that is still a bug that needs to be fixed... I'll have a look!
